src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py

```diff
@@ -759,7 +759,7 @@
 
 helps['postgres flexible-server replica stop-replication'] = """
 type: command
-short-summary: Stop replication to a read replica and make it a read/write server.
+short-summary: Stop replication to a read replica and make it a read/write server. This command is deprecated. Use 'az postgres flexible-server replica promote' instead.
```

Comment:
If you want to deprecate a command, pls modify the code in command definition. FYI https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments